pages/_app.tsx

```diff
@@ -3,18 +3,26 @@ import type { AppProps } from 'next/app'
 import { SessionProvider } from 'next-auth/react'
 import NavigationDrawer from '../components/NavigationDrawer'
 import HeaderAppBar from '../components/HeaderAppBar'
+import { useState } from 'react'
 
 export default function App({
   Component,
   pageProps: { session, ...pageProps },
 }: AppProps) {
+  const [navDrawerOpen, setNavDrawerOpen] = useState(false)
```

Comment:
i dont think this is necessary. These are going to be rendered in the layout once https://github.com/hack4impact-utk/community-coalition-against-human-trafficking/pull/4 is merged in

Reply:
Ah, my mistake, didn't realize we had something else to cover this. Removed.